### DISCUSSION TITLE
Where is the "Network" coming from in the window list?

### DISCUSSION THREAD

**Discussion by c02y** [davatorium/rofi#1348] [Q&A] [answered]
`rofi -show widow` will show all the opened windows, the problem is I noticed that every item in the list is started with the word "Network", I don't know where is the "Network" coming from, and apparently this word is wrong.

![image](https://user-images.githubusercontent.com/2135996/124627327-0850f380-deb2-11eb-9624-7855f2330d55.png)

There is no "Network" from the source code of rofi, there is no "Network" keyword from the desktop files for applications such as mpv, emacs and Thunar.

FYI:
v1.6.1
empty ~/.config/rofi

**Comment by DaveDavenport** [comment, marked as answer]
That is what your window manager reports as the name for the desktop. You can change what it prints (see rofi -h or rofi manpage.)

**Reply by c02y** [reply]
Oh, sorry, I renamed my XFCE workspace as the name "Network", thanks.
